package.json

```diff
@@ -1,115 +1,125 @@
 {
-  "name": "streamline",
-  "version": "0.0.0",
-  "license": "MIT",
+  "name": "@getluko/streamline",
+  "version": "0.0.53",
   "scripts": {
-    "prepare": "husky install",
-    "postinstall": "patch-package",
```

Comment:
These two can stay as is

Reply:
we do not use patch-package anymore.
For pre-commit, we use lefthook which came pre-configured with bob